src/LayerEditor/data/geometry_model.py

```diff
@@ -0,0 +1,325 @@
+"""Model for Layer Geometry File"""
```

Comment:
consistent file name

Reply:
Seems to mostly refactored GeometryFactory. Good way to go, just clearly distinguish unduable operations during operation and possible helper methods for serialization.